Add a configuration reference document for helm chart (#3569)
SumoLogic/sumologic-documentation: docs/send-data/kubernetes/configuration.md

---
id: configuration
title: Configuration
sidebar_label: Configuration
description: Sumo Logic Kubernetes Helm Chart configuration.
---

> Detailed list of Sumo Logic Kubernetes Helm Chart configurable parameters are available in [project repository](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/deploy/helm/sumologic/README.md)

Reviewer: @dmolenda-sumo do you plan to migrate [this page](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/deploy/helm/sumologic/README.md) to our Docs or will it be staying there?

Author: We have no plans to move this document. There is a script checks whether all fields in the config are explained, and we want to keep this feature.

> Detailed list of Sumo Logic Kubernetes Helm Chart configurable parameters are available in [project repository](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/deploy/helm/sumologic/README.md)

Reviewer: ```suggestion
See our [Sumo Logic Kubernetes Collection project repository](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/deploy/helm/sumologic/README.md) for a complete list of configurable Helm Chart parameters.
```
